src/Microsoft.AspNet.Mvc.TagHelpers/Internal/ModeAttributes`T.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
at least in MVC repo, convention does not use back-ticks.  so ModeAttributesOfTMode.cs or ModeAttributesOfT.cs